tutorials/quick-kubernetes.md

```diff
@@ -0,0 +1,16 @@
+# Quick kubernetes setup
+
+This tutorial assumes you've already installed CRI-O and have started it. If you have not, start [here](setup.md).
+
+You can run a local version of kubernetes with CRI-O using `local-up-cluster.sh`:
+
+1. Clone the [kubernetes repository](https://github.com/kubernetes/kubernetes)
+1. From the kubernetes project directory, run:
+```shell
+CGROUP_DRIVER=systemd \
+CONTAINER_RUNTIME=remote \
+CONTAINER_RUNTIME_ENDPOINT='unix:///var/run/crio/crio.sock  --runtime-request-timeout=15m' \
```

Comment:
I'm not sure if this `--runtime-request-timeout=15m` is needed, I never used it and it should work with a smaller timeout, right?